install_config/cfme/role_variables.adoc

```diff
@@ -0,0 +1,267 @@
+
+External PostgreSQL databases require you to provide database connection
+parameters. You must set the required connection keys in the
+`openshift_management_template_parameters` parameter in your inventory. The
+following keys are required:
+
+- `DATABASE_USER`
+- `DATABASE_PASSWORD`
+- `DATABASE_IP`
+- `DATABASE_PORT` (Most PostgreSQL servers run on port `5432`)
+- `DATABASE_NAME`
+
+[NOTE]
+====
+Ensure your external database is running PostgreSQL 9.5 or you may not be able
+to deploy the CloudForms application successfully.
+====
+
+Your inventory would contain a line similar to:
+
+----
+[OSEv3:vars]
+openshift_management_app_template=cfme-template-ext-db <1>
+openshift_management_template_parameters={'DATABASE_USER': 'root', 'DATABASE_PASSWORD': 'mypassword', 'DATABASE_IP': '10.10.10.10', 'DATABASE_PORT': '5432', 'DATABASE_NAME': 'cfme'}
+----
+<1> Set `openshift_management_app_template` parameter to `cfme-template-ext-db`.
+
+== Storage Class Variables
+
+[options="header"]
+|===
+|Variable |Required |Default |Description
+|`openshift_management_storage_class` 
+|No
+|`nfs` 
+|Storage type to use. Options are `nfs`, `nfs_external`, `preconfigured`, or
+`cloudprovider`. See
+xref:storage_class.adoc#install-config-cfme-storage-classes[Storage Class Options] for details on each.
+|`openshift_management_storage_nfs_external_hostname`
+|No 
+|`false` 
+|If you are using an external NFS server, such as a NetApp appliance, then you
+must set the host name here. Leave the value as `false` if you are not using
+external NFS. Additionally, external NFS requires that you create the NFS
+exports that will back the application PV and optionally the database PV.
+|`openshift_management_storage_nfs_base_dir`
+|No
+|`/exports/`
+|If you are using external NFS, then you can set the base path to the exports
+location here. For local NFS, you can also change this value if you want to
+change the default path used for local NFS exports.
+|`openshift_management_storage_nfs_local_hostname`
+|No 
+|`false` 
+|If you do not have an `[nfs]` group in your inventory, or want to simply
+manually define the local NFS host in your cluster, set this parameter to the
+host name of the preferred NFS server. The server must be a part of your
+{product-title} cluster.
+|===
+
+[[cfme-storage-class-nfs]]
+=== NFS (Default)
+
+The NFS storage class is best suited for proof-of-concept and test deployments.
+It is also the default storage class for deployments. No additional
+configuration is required for this choice.
+
+This storage class configures NFS on a cluster host (by default, the first master in
+the inventory file) to back the required PVCs. The application requires a PVC,
+and the database (which may be hosted externally) may require a second. PVC
+minimum required sizes are 5GiB for the {cloudforms} application, and 15GiB for
+the PostgreSQL database (20GiB minimum available space on a volume or partition
+if used specifically for NFS purposes).
```

Comment:
I just realized I misused the acronym `PVC` here in many places where it should be `PV` instead:

* back the required PV~C~s. 
* The application requires a PV~C~
* PV~C~ minimum required sizes are